modules/napimgui/src/imgui/imconfig.h

```diff
@@ -16,7 +16,7 @@
 
 //---- Define attributes of all API symbols declarations, e.g. for DLL under Windows.
 #ifdef _WIN32
-#ifdef NAP_SHARED_LIBRARY
+#ifdef NAP_SHARED_LIBRARY_IMGUI
```

Comment:
Unlike NAP_SHARED_LIBRARY, NAP_SHADER_LIBRARY_IMGUI is only defined in the mod_napimagui cmake list:

# preprocessors
target_compile_definitions(${PROJECT_NAME} PRIVATE NAP_SHARED_LIBRARY)
target_compile_definitions(${PROJECT_NAME} PRIVATE NAP_SHARED_LIBRARY_IMGUI)

Reply:
For completeness, here's the link to the discussion regarding this change: https://community.napframework.com/t/imgui-dll-export-issue/251

Reply:
So, after 2 years of fighting this, I'm accepting, reluctantly. Although the fix is relatively easy, it doesn't address the underlying issue. Since that's not our immediate fault I approve.